backend/route/login.route.js

```diff
@@ -4,14 +4,14 @@ const app = express();
 app.use(express.json());
 app.use(express.urlencoded({extended:false}))
 
+const checkLoginData=require('../middleware/login.middleware')
+const confirmLogin=require('../controller/login.controller')
+
 route.get('/', (req, res) => {
     console.log("Done my boi")
     res.send("done")
     res.end()
 })
-route.post('/',(req,res) => {
-    const name = req.body.name
-    res.send(name)
-})
+route.post('/',checkLoginData,confirmLogin)
```

Comment:
I like how you are handling this multiple middlewares O.O It is really cool.

Reply:
Check it now